src/Path.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file was imported from the webmozart/path-util package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Webmozart\Glob;
+
+/**
+ * Contains utility methods for handling path strings.
+ *
+ * The methods in this class are able to deal with both UNIX and Windows paths
+ * with both forward and backward slashes. All methods return normalized parts
+ * containing only forward slashes and no excess "." and ".." segments.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Thomas Schulz <mail@king2500.net>
+ *
+ * @internal this class was introduced to remove an external dependency, but is
+ *           not part of the public API of this package: do not use it directly.
+ */
+abstract class Path
```

Comment:
Abstract + final constructor? Why not just a function?

Reply:
Wanted to avoid a custom autoloader, which poses a performance penalty downstream.

Classes + static methods > functions, in PHP